packages/blocks/src/image-block/image-resize-manager.ts

```diff
@@ -77,7 +79,7 @@ export class ImageResizeManager {
     const rootComponent = getClosestRootBlockComponent(this._activeComponent);
     if (
       rootComponent &&
-      rootComponent.service.docModeService.getMode() === 'edgeless'
+      getMode(rootComponent.service.std) === DocMode.Edgeless
```

Comment:
I prefer to have APIs like `modeServcie.getMode()` rather than `getMode(rootComponent.service.std)`. The second one looks like a utility function.